addons/crm/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': 'CRM',
-    'version': '1.4',
+    'version': '1.5',
```

Comment:
As migration script is not necessary I think, no need to increase version number.

Reply:
Haha you're right, I did not pay attention to that, too used to migration process already.